src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -509,7 +508,7 @@ class CheatsWindow final : public Window
         // Current weather
         window_cheats_misc_widgets[WIDX_WEATHER].text = WeatherTypes[EnumValue(gClimateCurrent.Weather)];
         // Staff speed
-        window_cheats_misc_widgets[WIDX_STAFF_SPEED].text = _staffSpeedNames[_selectedStaffSpeed];
+        window_cheats_misc_widgets[WIDX_STAFF_SPEED].text = _staffSpeedNames[static_cast<int>(gCheatsSelectedStaffSpeed)];
```

Comment:
```suggestion
        window_cheats_misc_widgets[WIDX_STAFF_SPEED].text = _staffSpeedNames[ToUnderlying(gCheatsSelectedStaffSpeed)];
```

If `gCheatsSelectedStaffSpeed` is an enum, we have this convenience function to get the int.

Reply:
@tupaschoal - please elaborate. 

Do you mean std::to_underlying?
The other candidate could be ToUnderlying() from Identifier.hpp that's for class TIdentifier and returns a static cast anyway.

Reply:
We have a function called EnumValue which is doing the same as std::to_underlying, I think he mixed it up. It is defined in util/Util.h

Reply:
Whoops, @ZehMatt is right, sorry for the confusion!

Reply:
> We have a function called EnumValue which is doing the same as std::to_underlying, I think he mixed it up. It is defined in util/Util.h

Thanks, done!